package.json

```diff
@@ -71,7 +71,7 @@
     "replace": "^1.2.0",
     "sinon": "^2.4.1",
     "sinon-chai": "^2.14.0",
-    "three": "^0.123.0",
+    "three": "^0.125.0",
```

Comment:
We probably want to test against aframe ^1.3.0 and three ^0.137.0.

Reply:
Yes, sensible.  I was going for the minimum required change here, but if I don't introduce new issues upgrading further I'll do what you suggest. (and if I do, I guess we'll need to fix them anyway).

Reply:
Yes, and if you can, can you please test the examples with an aframe master build as well? aframe master is currently using three r144, you may catch some warnings to be fixed.

Reply:
OK, so the test issue that I hit here: https://github.com/c-frame/aframe-physics-system/pull/4#issuecomment-1264599504 turns out to appear whenever the tests use A-Frame > 1.1.0.

I don't understand this but moving tests to A-Frame 1.2.0+ breaks them as the browserify build step fails.

I'll raise a separate issue for that, but for now it's better to have tests that run cleanly against 1.1.0 than tests that don't work at all against 1.2.0 or 1.3.0.

That said it is a little moot as the tests barely cover anything anyway...!

Reply:
FYI aframe 1.1.0 is using super-three 0.123.1 so it's Quaternion.inverse.
aframe 1.2.0 is using super-three 0.125.0 so it's Quaternion.invert. so aframe 1.2.0 should be the minimum we support here. But yeah because the tests are not covering much of the code, it probably doesn't fail with aframe 1.1.0. :)
I'm ok with keeping 1.1.0 here for now in this PR, we should probably move from browserify to webpack anyway in a separate PR like I did with aframe and networked-aframe.

Reply:
Separate issue raised here:
https://github.com/c-frame/aframe-physics-system/issues/5

If moving to webpack will make this obsolete that sounds like a good option too.